local-cli/install/install.js

```diff
@@ -10,7 +11,17 @@ log.heading = 'rnpm-install';
 function install(args, config) {
   const name = args[0];
 
-  var res = spawnSync('npm', ['install', name, '--save'], spawnOpts);
+  const projectDir = config.getProjectRoots()[0];
```

Comment:
Is `config.getProjectRoots()` guaranteed to return an array with at least one element? What if it doesn't, or what if it starts returning `undefined` under some circumstances in the future? Better check for it just to be safe.

Is this the standard we use to get the current project dir everywhere else in the CLI, or are there other ways that might be more reliable? Is it possible that people override `getProjectRoots` in their config to return something else than the folder you'd actually want to use here in the `isGlobalCliUsingYarn` check? The `projectDir` here should always be something like `~/MyApp`, the same folder where package.json lives, correct?

Just asking some basic questions, I'm not familiar with how the config works.